cloudsql-mysql-plugin/src/main/java/io/cdap/plugin/cloudsql/mysql/CloudSQLMySQLAction.java

```diff
@@ -103,6 +112,10 @@ public String getConnectionString() {
           connectionName);
     }
 
+    public Integer getPort() {
```

Comment:
need to be careful with how this is used though. If the port is a macro, it will be null at configure time and this will return 3306. If we do any connectivity validation at configure time, it will need to be skipped if the port is a macro, otherwise it will try to connect on the wrong port.

Reply:
I have overridden the `canConnect()` method which return false if any of the connection property is macro.

Does it makes sense? wdyt?